sidechain/consensus/aura/src/slot_proposer.rs

```diff
@@ -147,6 +148,13 @@ impl<ParentchainBlock, SignedSidechainBlock, TopPoolExecutor, BlockComposer, Stf
 
 		parentchain_extrinsics.push(confirmation_extrinsic);
 
+		info!(
+			"Queue/Timeslot/Transactions: {:?};{};{}",
```

Comment:
Maybe this can be improved by putting the number close to the description:
`Queue length: {}, Slot duration: {}, Number of transactions executed: {}`